api-documentation.md

```diff
@@ -470,12 +471,32 @@ window.onmessage = function(e){
 }
 ```
 ```js
+// currently used by old integrations, use surveyCompletedWithData instead
```

Comment:
I would add the comment above the `if(e.data)...` line since now it is kind of confusing